src/sdc11073/mdib/descriptorcontainers.py

```diff
@@ -244,55 +239,64 @@ def sort_child_nodes(self, node: etree_.Element) -> None:
                     node.append(child_node)
 
     def set_source_mds(self, handle: str):
```

Comment:
if get and set methods are public, why not just making `self._source_mds` public? -> `self.source_mds`

Reply:
Because this allows me to set a breakpoint and see where the value came from.